slides/02_Setup.html

```diff
@@ -204,45 +219,77 @@ <h2>NPM scripts</h2>
         <section>
           <section>
             <h2>ESLint</h2>
-            <p>Enforce some code styles which pertain more to usage.</p>
-            <p>Create React App comes with some predefined rules.</p>
+
             <p>
-              ESLint has a ton of (predefined) rules, ranging from very loose to
-              super strict (<a
-                href="https://www.npmjs.com/package/eslint-config-airbnb"
+              ESLint is a tool that statically analyzes your code to discover
+              potential problems (<span class="warn">linting</span>).
+            </p>
+            <p class="fragment">
+              Create React App comes with ESLint <em>built-in</em>. Problems are
+              shown in the terminal.
+            </p>
+            <p class="fragment">
+              Recommendation: Install the
+              <a
+                href="https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint"
                 target="_blank"
                 rel="noopener noreferrer"
-                >airbnb</a
-              >).
+              >
+                VS Code plugin for ESLint
+              </a>
+              for highlighting and fix assistance in the editor.
             </p>
```

Comment:
Additional Tip: Run Lint & Unit Tests as part of the CI/CD pipeline.

Reply:
Good idea. I added two slides: one general regarding CI in the section of the "npm scripts", and one specifically regarding linting on the CI (explaining that npm run build treats warnings as errors on CI).